src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -1282,6 +1283,48 @@ DB2StorageBase const* DB2Manager::GetStorage(uint32 type) const
     return nullptr;
 }
 
+void DB2Manager::LoadHotfixBlob()
+{
+    uint32 oldMSTime = getMSTime();
+
+    QueryResult result = HotfixDatabase.Query("SELECT TableHash, RecordId, Size, Blob FROM hotfix_blob ORDER BY TableHash");
+
+    if (!result)
+    {
+        TC_LOG_INFO("server.loading", ">> Loaded 0 hotfix blob entries.");
+        return;
+    }
+
+    uint32 count = 0;
+    do
+    {
+        Field* fields = result->Fetch();
+
+        uint32 tableHash = fields[0].GetUInt32();
+
+        HotfixBlobData blobData;
+        blobData.RecordId = fields[1].GetInt32();;
+        blobData.Size = fields[2].GetUInt32();;
+        blobData.Blob = fields[3].GetBinary();;
```

Comment:
I may be missreading this, please help me: Why is a vector needed if you don't need the different indexes?

Reply:
ReadBinary Output is vector

Reply:
Ahhhh... thanks